### ISSUE TITLE
No headings, button used instead of link

### ISSUE THREAD

**Issue by aardrian** [bpetetot/conference-hall-beta#857]
On the page: https://conference-hall.io/public/event/FdP7ePMBrbLq9kZd4Hye

* The heading structure is unexpected. "AccessibilityCon", sitting over the hero image, should be an `<h1>`, and "Call for paper is opened" should be an `<h2>`. Instead, the page has no `<h1>` nor `<h2>` and the first headings are "Talk categories" and "Talk formats" each as `<h3>` (correct if the other headings are added).
* The "Submit a Talk" control is a `<button>`, which is fine for changing something on the page (displaying content, toggling a box, popping a modal). It is used to navigate the user to a new page, however, so it should be a link.

For more details and guidance, see:
* WCAG 2.1 Success Criterion [1.3.1 Info and Relationships, Level A](https://www.w3.org/WAI/WCAG21/quickref/#info-and-relationships);
* [Technique H42: Using h1-h6 to identify headings](https://www.w3.org/WAI/WCAG21/Techniques/html/H42.html);
* WCAG 2.1 Success Criterion [4.1.2 Name, Role, Value, Level A](https://www.w3.org/WAI/WCAG21/quickref/#name-role-value);
* [Technique H91: Using HTML form controls and links](https://www.w3.org/WAI/WCAG21/Techniques/html/H91.html).

.

**Comment by bpetetot** [comment]
Thanks again for the feedback 😉 🙏

As you recommend, we applied a better heading structure for the page.
Fix: ee74ee9fa7fa1349f31839a08aae26e690f8ef36

Submit talk has been replaced by a link (instead of a button)
Fix: 3a5f6fe6b132928cf59500cfc4facce348592c25
